src/main/java/client/Stat.java

```diff
@@ -76,44 +76,10 @@ public static Stat getBy5ByteEncoding(int encoded) {
     }
 
     public static Stat getByString(String type) {
-        if (type.equals("SKIN")) {
-            return SKIN;
-        } else if (type.equals("FACE")) {
-            return FACE;
-        } else if (type.equals("HAIR")) {
-            return HAIR;
-        } else if (type.equals("LEVEL")) {
-            return LEVEL;
-        } else if (type.equals("JOB")) {
-            return JOB;
-        } else if (type.equals("STR")) {
-            return STR;
-        } else if (type.equals("DEX")) {
-            return DEX;
-        } else if (type.equals("INT")) {
-            return INT;
-        } else if (type.equals("LUK")) {
-            return LUK;
-        } else if (type.equals("HP")) {
-            return HP;
-        } else if (type.equals("MAXHP")) {
-            return MAXHP;
-        } else if (type.equals("MP")) {
-            return MP;
-        } else if (type.equals("MAXMP")) {
-            return MAXMP;
-        } else if (type.equals("AVAILABLEAP")) {
-            return AVAILABLEAP;
-        } else if (type.equals("AVAILABLESP")) {
-            return AVAILABLESP;
-        } else if (type.equals("EXP")) {
-            return EXP;
-        } else if (type.equals("FAME")) {
-            return FAME;
-        } else if (type.equals("MESO")) {
-            return MESO;
-        } else if (type.equals("PET")) {
-            return PET;
+    	for (Stat stat : Stat.values()) {
+            if (stat.name().equals(type)) {
+                return stat;
+            }
```

Comment:
Good improvement, but I can't find any usages of this function at all. Can be removed altogether.